config/initializers/carrierwave.rb

```diff
@@ -27,7 +27,8 @@ def self.standard_production_config
         provider: "AWS",
         aws_access_key_id: ApplicationConfig["AWS_ID"],
         aws_secret_access_key: ApplicationConfig["AWS_SECRET"],
-        region: ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"]
+        region: ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"],
+        endpoint: ApplicationConfig["AWS_S3_ENDPOINT"],
```

Comment:
if you're adding new environment variables, setting this to a reasonable default in .env_sample is polite :)

Reply:
@djuber The default is `nil`, which falls back to using S3 as if we didn't provide `endpoint` at all (the current behavior). Should we still add it to the sample config?

Reply:
I think it should still be added to the sample config set to blank because there's value in listing all AWS-related keys that can be utilized.

Reply:
@maestromac That's fair. Would we need to adjust these to use `.presence` in that case? Will `dotenv` give us the empty string if we provide a key but no value?

Reply:
Yes `.presence` is needed. I just tried the following
```
ApplicationConfig["AWS_UPLOAD_REGION"]
=> ""
```

Reply:
@maestromac Hmm, I don't love that. If we're providing the key, it seems like we should use it. `Object#presence` is primarily useful for when an empty string means someone didn't provide a value, such as in an HTML form because the user can't control whether the form input's value is sent to the server. Apart from the fact that I'm too lazy and stubborn to add `.presence` everywhere, expecting everything in the entire application to adapt to that equivalence every single time they read an env var sounds like a lot of bugs waiting to happen (and IIRC the `.presence above this line is the result of one of them).

In the case of environment variables, though, we _do_ control whether the env var is provided. Maybe we should comment them out the way config files in a lot of other projects do? Or, as a smaller stepping stone, instead of having to treat `""` and `nil` the same _everywhere_, duplicating a lot of effort, we could instead have an initializer do something like `ENV.reject! { |key, value| value == "" }` to normalize the values to `nil`?

Reply:
Yes! I agree with you completely. I vote for just commenting them out for now